src/data/actions/licenseRevoke.js

```diff
@@ -0,0 +1,47 @@
+import {
+  LICENSE_REVOKE_REQUEST,
+  LICENSE_REVOKE_SUCCESS,
+  LICENSE_REVOKE_FAILURE,
+} from '../constants/licenseRevoke';
+
+import EcommerceApiService from '../services/EcommerceApiService';
```

Comment:
I don't think we should be using `EcommerceApiService` here? The license-manager service will be the one to call for anything related to licenses.

https://github.com/edx/frontend-app-admin-portal/blob/master/src/components/subscriptions/data/service.js